app/src/main/res/layout/help_item.xml

```diff
@@ -13,7 +13,7 @@
     android:id="@+id/help_item_text_view"
     style="@style/Subtitle1ViewStart"
     android:layout_width="match_parent"
-    android:background="@drawable/general_item_background_border"
```

Comment:
Use `general_item_border` instead of `general_item_background_border `

and `general_item_border_cyan` instead of `general_item_background_border_cyan`

Reply:
Done, divider issue is fixed now.